api/libproject/devstart.rb

```diff
@@ -63,6 +63,7 @@ def read_db_vars(gcc)
     Common.new.error "Failed to read #{cdr_vars_path}"
     exit 1
   end
+  vars["GOOGLE_APPLICATION_CREDENTIALS"] = "sa-key.json"
```

Comment:
This must be present when connecting to Cloud SQL without the Cloud SQL Proxy.

Reply:
yes this does seem to fix me.  thanks!